packages/expo-video/android/build.gradle

```diff
@@ -21,7 +21,7 @@ android {
 dependencies {
   implementation 'com.facebook.react:react-android'
 
-  def androidxMedia3Version = "1.3.1"
+  def androidxMedia3Version = "1.4.0"
```

Comment:
A comment to try to keep it in sync with `expo-audio` might be useful
cc @alanjhughes